index.html

```diff
@@ -9,7 +9,7 @@
       <ul class="clearfix">
         {% for post in paginator.posts %}
           <li>
-            <a href="{{ post.url | prepend: site.url }}">
+            <a href="{{ post.url | prepend: site.baseurl }}">
```

Comment:
I'm deliberately setting this to be a relative URL, so that links on the test/staging server don't point to production.